packages/ui/src/Customization/models.ts

```diff
@@ -0,0 +1,27 @@
+// Extension Types
+export enum ExtensionType {
+  Connectors = 'Connectors',
+  Libraries = 'Libraries',
+  Steps = 'Steps',
+}
+
+export interface IAction {
```

Comment:
This interface and `IIntegration` feel like types that we don't need to define, at least in this way, *however* I don't yet have a better suggestion at the moment :-)